reopt_api/staging_settings.py

```diff
@@ -112,7 +112,7 @@
     'default':{
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'HOST': staging_database_host,
-        'NAME': staging_database_name,
+        'NAME': 'reopt_staging',
```

Comment:
@Bill-Becker just a reminder that we need to switch this staging setting back to use the keys.py value at some point

Reply:
I changed it back to the keys.py value.